Automattic/wp-calypso
# Reader: move followed sites to left sidebar

Relates to Automattic/loop#190

## Proposed Changes

* Move the followed sites list from the right sidebar to under "Recent" in the left sidebar
* Truncate the list of sites to 8 if there are more than 8
* Clicking a site filters the posts in the middle recent column to just that site (this PR just wires up the state; the integration with the UI will come in another PR)

### Does not address (will come in another PR)

* Styling
* Mobile
* Empty state
* "Unseen" counts. See Automattic/loop#216
* Accessible keyboard navigation

## Why are these changes being made?


* pe7F0s-2iv-p2

## Testing Instructions



* Go to /read?flags=reader/recent-feed-overhaul
* Verify that your followed sites show up in the Recent section on the left
* Verify that only 8 sites show up by default
* If there are more than 8 sites, a "view more" button should be present
* Clicking the "view more" button should expand to show all sites
* Clicking "view less" should collapse back to 8 sites
* Verify the sites are sorted by most recent post first (the top of the list)
* If you can inspect Redux via devtools, verify that clicking each site updates the `selectedRecentSite` reader ui state to the selected `feedId` or `null` if "All" is selected

## Pre-merge Checklist
